src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Queries/GetWorkplacesQuery.g.cs

```diff
@@ -8,33 +8,64 @@
 using Nox.Application.Commands;
 
 using ClientApi.Application.Dto;
-using ClientApi.Infrastructure.Persistence;
+using ClientApi.Infrastructure.Persistence;using Nox.Presentation.Api;
+using Nox.Solution;
 
 namespace ClientApi.Application.Queries;
 
 public record GetWorkplacesQuery() : IRequest<IQueryable<WorkplaceDto>>;
 
 internal partial class GetWorkplacesQueryHandler: GetWorkplacesQueryHandlerBase
 {
-    public GetWorkplacesQueryHandler(DtoDbContext dataDbContext): base(dataDbContext)
+    public GetWorkplacesQueryHandler(DtoDbContext dataDbContext,
```

Comment:
Quesries are Application Level, they can not depend on HTTP function, they can be invoked by any component, for example a IntegrationEvent Consumer

The query needs to receive a Nox.Type.CultureCode

Reply:
Done